R/get.R

```diff
@@ -82,12 +82,20 @@ get_file_path <- function(file = NA, normalize = TRUE){
 #' @export
 #'
 #' @importFrom sessioninfo session_info
+#' @importFrom purrr map
+#' @importFrom stringr str_wrap
 #'
 #' @examples
 #' get_session_info()
 #'
 get_session_info <- function(){
```

Comment:
This update might cause some formatting issues after the wrap.  I noticed the .log was slightly different between the two times I have run .  
![image](https://user-images.githubusercontent.com/10111024/160899038-4c1c0394-0426-4fea-8ecb-e2ffd58a9c73.png)